processor.go

```diff
@@ -321,6 +372,7 @@ func (p *processor) send(clientIP net.Addr, reqID uuid.UUID, tenant string, wr *
 		return
 	}
 
+	duration = float64(time.Since(start).Milliseconds())
```

Comment:
I guess since we anyway are using `float64` then it's better to store seconds there, not milliseconds.

Reply:
This was made milliseconds because it's more likely for this value to be pretty small... I ripped this metric and buckets from the tool Istio